service-workers/service-worker/resources/fetch-event-handled-worker.js

```diff
@@ -0,0 +1,42 @@
+// This worker reports back the final state of FetchEvent.handled (RESOLVED or
+// REJECTED) to the test.
+
+// Send a message to all clients including those not controlled.
+function send_message_to_all_clients(message) {
+  clients.matchAll({includeUncontrolled: true}).then((allClients) => {
```

Comment:
I think it would be better to use the FetchEvent.clientId to only send the message to the frame that invoked the fetch().  That way if WPT runs tests in parallel you are not spamming other test frames with unexpected messages.

Reply:
Done, thanks for the suggestion.